examples/docker/docker-compose-full.yml

```diff
@@ -0,0 +1,64 @@
+# This docker-compose file starts an entire wasmCloud ecosystem, including:
+#   a NATS server
+#   a local OCI registry
+#   grafana + tempo for tracing
+#   a wasmCloud host
+#   a WADM server for managing applications
+
+version: "3"
+services:
+  nats:
+    image: nats:2.10.7-alpine
+    ports:
+      - "4222:4222"
+      - "6222:6222"
+      - "8222:8222"
+    command: ["-js"]
+  registry:
+    image: registry:2.8
+    ports:
+      - "5000:5000"
+  grafana:
+    image: grafana/grafana:10.0.10
+    ports:
+      - 5050:3000
+    volumes:
+      - ./grafana-datasources.yaml:/etc/grafana/provisioning/datasources/datasources.yaml
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+    depends_on:
+      - tempo
+  tempo:
+    image: grafana/tempo:2.3.1
+    command: ["-config.file=/etc/tempo.yaml"]
+    volumes:
+      - ./tempo.yaml:/etc/tempo.yaml
+    ports:
+      - 8000:8000 # tempo
+      - 55681:55681 # otlp http
+
+  wasmcloud:
+    depends_on:
+      - "nats"
+      - "grafana"
+      - "tempo"
+    image: wasmcloud/wasmcloud:0.81.0-rc1 # TODO(nomerge!): update back to latest
+    environment:
+      RUST_LOG: debug,hyper=info,async_nats=info,oci_distribution=info,cranelift_codegen=warn
+      WASMCLOUD_LOG_LEVEL: debug
+      WASMCLOUD_RPC_HOST: nats
+      WASMCLOUD_CTL_HOST: nats
+      WASMCLOUD_OCI_ALLOWED_INSECURE: registry:5000
+      OTEL_TRACES_EXPORTER: otlp
+      OTEL_EXPORTER_OTLP_ENDPOINT: http://tempo:55681/v1/traces
+    ports:
+      - "8080-8089:8080-8089" # Allows exposing examples on ports 8080-8089
```

Comment:
Should we be more explicit about what it's using these ports? May not be clear that examples *running httpservers* are the most likely thing to use those ports.

Reply:
I raised an eyebrow at this too (and briefly got confused when I tried to link the HTTP server on port 1234 and requests failed)

What do you think we should do? Just improve the comment by clarifying these are for examples using the HTTP server? Widen the range? Something else?

Reply:
Yeah I think for now improving the comment is the easiest thing... We clearly don't want to expose *all* the ports.

Isn't the default for httpserver 8000? Maybe we should include that one too, though I know often examples use 8081. It might be reasonable to take 8000-8100 or something.

Reply:
Yeah I think for now improving the comment is the easiest thing... We clearly don't want to expose *all* the ports.

Isn't the default for httpserver 8000? Maybe we should include that one too, though I know often examples use 8081. It might be reasonable to take 8000-8100 or something.

Reply:
Yeah the default is 8000. 8000-8100 sounds good to me 👍